ex1/functions.py

```diff
@@ -46,15 +46,66 @@ def f():
 ##############################
 
 # Define a function named add that takes two numbers and returns the sum.
+def add(num1, num2):
+	"""
+	Parameters
+	----------
+	num1, num2 : int
+
+	Returns 
+	-------
+	int	
+		sum of num1 and num2
+    	"""
+	return num1+num2
 
 # Define a function named to_tuple that takes three arguments and returns a
 # tuple of these three arguments.
+def to_tuple(a1, a2, a3):
+	"""
+	Parameters
+	----------
+	a1, a2, a3 : optional
+
+	Returns 
+	-------
+	tuple of objects
+			with 3 Objects. Objects := Parameters
+			order: {a1, a2, a3}
+    	"""
+	return (a1,a2,a3)
```

Comment:
You do not need the brackets here since multiple return values are automatically packed into a tuple.